infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -457,13 +471,12 @@ private synchronized void detectorTaskScheduler() {
      * @return True if node is allowed to handle reconfigurations. False otherwise.
      */
     private boolean canHandleReconfigurations() {
-
         // We check for the following condition here: If the node is NOT a part of the
         // current layout, it should not attempt to change layout.
+
         Layout layout = serverContext.getManagementLayout();
-        if (!layout.getAllServers().contains(getLocalEndpoint())) {
-            log.debug("This Server is not a part of the active layout. "
-                    + "Aborting reconfiguration handling.");
+
+        if (!layout.getAllServersNodes().contains(getLocalEndpoint())) {
```

Comment:
it should be getAllServers or getAllServerNodes

Reply:
fixed

Reply:
I mean now, alomost everywhere we use getAllServerNodes